src/Makefile

```diff
@@ -3,17 +3,25 @@ PLATFORM_DIR = platforms/$(PROBE_HOST)
 VPATH += $(PLATFORM_DIR) target
 ENABLE_DEBUG ?=
 
+CC ?= gcc
+SYS = $(shell $(CC) -dumpmachine)
+
 ifneq ($(V), 1)
 MAKEFLAGS += --no-print-dir
 Q := @
 endif
 
-CFLAGS += -Wall -Wextra -Werror -Wmaybe-uninitialized \
-	-Wreturn-type -Wstringop-overflow -Wunsafe-loop-optimizations \
+CFLAGS += -Wall -Wextra -Werror \
```

Comment:
Please reflow `-Wreturn-type` back onto this line

Reply:
Done